### ISSUE TITLE
Exception in netty pipeline console error spam

### ISSUE THREAD

**Issue by Defflix** [MehVahdJukaar/FarmersDelightRefabricated#113]
### Minecraft version

1.21.1

### Farmer's Delight version

2.2.2

### Fabric Loader version

0.16.9

### Fabric API version

0.108.0+1.21.1

### Description

When using the Farmer's Delight Refabricated mod, it produces a really annoying console spam that overshadows any other messages in logs. I've also found out that it probably overwrites the beginning part of the log, since when I download the log made overnight, it cuts off at the start with only errors being present. 
Now, I use a pretty specific server setup. I run my server on Oracle Cloud Infrastructure free ARM tier, with 4 ARM Ampere Altra cores and 12GB of ram with Aikar's flags. I have ports 25565 and 24454 opened, one for the game port and the second one for Voice Chat (I also set up a test server with only 1 port, 25566, opened and it still occurred). 
Another part of this spam is that although it occurs when running Farmer's Delight Refabricated on it's own, adding more mods seems to speed it up. When I'm running a server with my whole modpack, the error occurs much faster than running Farmer's Delight Refabricated on it's own. I usually have to wait overnight for the error spam to show up in the second case. I'm almost certain this spam is caused by Farmer's Delight, since removing it from my full modpack and letting it run for over 24 hours didn't produce any spam.

This spam is extremely annoying and I've been dealing with figuring out which mod does it for like 2 weeks. If this is an error on my end (extra port needs to be opened...) then please tell me. 

### Steps to reproduce

1. Set up a Minecraft server on OCI free ARM tier
2. Create a server
3. Run Farmer's Delight Refabricated and its dependencies
4. Wait (usually takes a few hours when running barebones)

### Mod list

I've tried this on a test server with numerous mod combinations, and found that it still occurs with only **Farmer's Delight** (and it's dependency, **Fabric API**) present. Here is the full modpack I use on my server: https://api.mclo.gs/1/raw/BTa6HWs

### Logs

Here are 2 logs: the shorter one is in a very minimal instance, with only Farmer's Delight, Comforts and AdventureZ. The longer one is with my full modpack present. 
https://api.mclo.gs/1/raw/VIKnmRv
https://api.mclo.gs/1/raw/63mae2U

### Minimal instance

- [X] I have tested this on a minimal instance

### Performance and shader mods

- [ ] I am using performance or shader mods

**Comment by AmeriBaha** [comment]
I have the same problem

![image](https://github.com/user-attachments/assets/4104f02f-028e-4717-b94e-43953d139316)

**Comment by Pressburg** [comment]
I've made a thread in the Fabric discord server, here's the link: https://discord.com/channels/507304429255393322/1309586202340622446
It might or might not be useful. The conversation boils down to this:
- When ingress is disabled on the server (the game port is closed in OCI) the error doesn't happen anymore.
- At first, we thought the cause must be server scanning, but now I'm quite confident it isn't. I tried a mod called Sneaky (https://modrinth.com/mod/sneaky-server) which is supposed to hide your server from scanners (confirmed working, can't be pinged from outside, only sends ping info to previously joined players) and the error still occurs.

**Comment by ChrysanthCow** [comment]
Nothing we do **should** be causing this. As the person in the Fabric Discord server said, it's purely coincidence that it tries joining when this mod is on your client.

I promise that this mod isn't malware.

**Comment by ChrysanthCow** [comment]
Where did you download the mod, and what version?

**Comment by Pressburg** [comment]
> Where did you download the mod, and what version?

I downloaded the mod either from Modrinth or CurseForge. I used 2.2.2 in the beginning, but it happens on 2.2.3 too (the version I’m using right now). 

Also, maybe ignore my last comment. I’m too lost in this to draw conclusions. 
![IMG_1670](https://github.com/user-attachments/assets/add8137e-e74a-4dfd-b24d-72ad707d29ed) Here’s a photo where [osrond](https://2b2t.miraheze.org/wiki/Orsond) tried to join the server (probably actually them since they make server scanners). I can dig up the log where this occured when I come home.

I’ll also try removing Farmer’s Delight Refabricated again on a test server and see if anything changed.

**Comment by Pressburg** [comment]
The hunt for this bug has been insane. I still don't know its cause, but its very likely a combination of a [Minecraft bug](https://bugs.mojang.com/browse/MC-271325) and server scanners. 

When removing Farmer's Delight from the modpack, the error still appeared, so my initial research must have been wrong.
What I did to fix the issue:

- Added [Disconnect Packet Fix](https://modrinth.com/mod/disconnect-packet-fix) to my modpack. Didn't seem to directly fix the issue, but I still have it there just to be sure. 
- Added [Sneaky](https://modrinth.com/mod/sneaky-server) to my modpack. I use it on its default settings, and it seems to work. 
- Most importantly, I moved my game port from 25565 to a random one.
